Week8 - homework SQL/SQLQuery_2.sql

```diff
@@ -0,0 +1,157 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE TicketTypes
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Subscription VARCHAR(20),
+Price INT,
+Validity VARCHAR(20),
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+LastName VARCHAR(50),
+FirstName VARCHAR(50),
+TicketId INT REFERENCES TicketTypes(Id),
+EntryDate Date,
+);
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(50),
+);
+
+CREATE TABLE Animals
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(50),
+Species INT REFERENCES Species(Id),
```

Comment:
SpeciesId would be a better name for the foreign key as you did above with TicketId - following the same naming convention everywhere makes the database easier to understand.